.storybook/main.js

```diff
@@ -31,4 +53,7 @@ module.exports = {
       },
     };
   },
+  docs: {
+    autodocs: false,
```

Comment:
can we add a comment here to document why this is set to false? Thank you!

Reply:
Sure. 🥸
Yes, the new storybook v7 includes a Docs tab in addition to the Default, Live preview, and test stories. I set it to false because we don't want it.